### PR TITLE
add service configuration and default config

### PR DESCRIPTION
**Pull request by sandrokeil**
Please don't merge. It's not finished yet.

### CODE REVIEW

**Comment by sandrokeil** [comment]
@codeliner Updated readme. We have to tag version 0.1 of [prooph-cli](https://github.com/proophsoftware/prooph-cli) before release.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+          # list of commands with corresponding command handler
+          'Prooph\ProophessorDo\Model\User\Command\RegisterUser': 'Prooph\ProophessorDo\Model\User\Handler\RegisterUserHandler'
+    event_bus:
+      router:
+        routes:
+          # list of events with a list of projectors
+          'Prooph\ProophessorDo\Model\User\Event\UserWasRegistered':
+            - 'Prooph\ProophessorDo\Projection\User\UserProjector'
+  event_store:
+    # list of event stores
```
Comment (codeliner) [inline README.md]: Should be list of aggregate repositories

File: README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    </services>
+</container>
+```
+
+Here is an example how to call the `RegisterUser` command:
+
+```php
+    /* @var $container \Symfony\Component\DependencyInjection\ContainerBuilder */
+    $commandBus = $container->get('prooph.service_bus.command_bus');
+    $messageFactory = new \Prooph\Common\Messaging\FQCNMessageFactory();
```
Comment (codeliner) [inline README.md]: We should not use the message factory here as you can instantiate the command by simply doing: `$command = new \Prooph\ProophessorDo\Model\User\Command\RegisterUser([/*payload here*/])`

= less code, easier to read

**Comment by codeliner** [comment]
@sandrokeil well done! only minor remarks. [wip] can be removed? prooph-cli is tagged with v0.1

**Comment by sandrokeil** [comment]
@codeliner Alright. Updated docs. Ready to merge

**Pull request merged**